src/utility/verification/verify-resource.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import { Probot } from 'probot';
+import { Resource } from '../../service/resource/resource';
+
+export async function verifyOrgRepo(app: Probot, context: any, resource: Resource): Promise<boolean> {
```

Comment:
NIT: Can we rename this `validateResourceConfig` or use `getRepo`, `getOrg` and then match with context repo and org.

Reply:
Will check, thanks.

Reply:
Just did a check and since I change all the getters to use proper `get` instead of `getSomething` func, `resource.organization` is already effectively using the getter, same as `getOrganization` before.

I go ahead and changes the name to `validateResourceConfig` so we can extend this with more checks later.

Thanks.